app/assets/javascripts/reducers/users.js

```diff
@@ -6,6 +6,7 @@ const initialState = {
   sort: {
     sortKey: null,
     key: null,
+    initialSortKey: 'real_name'
```

Comment:
Can this value not just be used for sortKey on line 7? then there's no need for conditionals within the reducer.

Reply:
Hmm, I didn't think of it this way. This definitely makes more sense and much cleaner code! Tried and it works!